utils/tokenAssertions.ts

```diff
@@ -22,6 +22,15 @@ export async function expectRevertError(asyncTxn: any) {
   }
 }
 
+// For solidity function calls that do not violate require()
+export async function expectNoRevertError(asyncTxn: any) {
```

Comment:
With test cases that pass, I expect there to be an `expect` or test case.

In this case, there is no test case in the try block. Maybe something like expect(true) would suffice if the `await asyncTxn` passes